android/src/main/kotlin/com/bitmovin/player/flutter/FlutterPlayerView.kt

```diff
@@ -111,6 +119,28 @@ class FlutterPlayerView(
         sink = null
     }
 
+    private fun PlayerView.setOnPictureInPictureModeChanged(callback: (Boolean, Configuration) -> Unit) {
+        var isInPictureInPictureMode = activity.isInPictureInPictureMode
+        // Listening to PiP changes usually happens by overriding onPictureInPictureModeChanged in the Activity.
+        // This is not really doable in a SDK context since the library consumer actually controls the activity.
+        // To work around this limitation we listen to configuration changes and check if the PiP mode changed.
+        // Since Flutter's PlatformView isn't actually an Android View,
+        // we unfortunately cannot just override onConfigurationChanged function.
+        // Instead, we add a blank View to the PlayerView and override its onConfigurationChanged function.
+        // The alternative would be registering a ComponentCallbacks object to the activity, which unfortunately doesn't
+        // work for some Android versions (tested on Android 10 with FlutterFragmentActivity).
```

Comment:
NIT, no blocker
```suggestion
        // Listening to PiP changes usually happens by overriding `onPictureInPictureModeChanged` in the activity.
        // This is not doable in an SDK context since the library consumer controls the activity.
        // To work around this limitation we listen to configuration changes and check if the PiP mode changed.
        // Since Flutter's `PlatformView` isn't actually an Android `View`,
        // we unfortunately cannot just override the `onConfigurationChanged` function.
        // Instead, we add a blank `View` to the `PlayerView` and override its `onConfigurationChanged` function.
        // The alternative would be registering a `ComponentCallbacks` object to the activity, which unfortunately doesn't
        // work for some Android versions (tested on Android 10 with FlutterFragmentActivity).
```